documentation/reference/grouping-syntax.html

```diff
@@ -381,247 +380,6 @@ <h2 id="expression">Expressions</h2>
 
 
 
-    <h2 id="examples">Examples</h2>
```

Comment:
The example section contains a more detailed description of some of the operators. I agree on moving the examples, but we should consider keeping the operator descriptions in this document.

Reply:
E.g first section of _Counting unique groups_

Reply:
agreed! I will add back to reference guide what are not strictly examples, easier to do not as the reference is more like a reference

Reply:
I will do that in a separate PR